src/react/workflow/replication/Replication.jsx

```diff
@@ -28,6 +28,10 @@ const ReplicationContainer = styled.div`
     }
 `;
 
+const ReplicationIcon = styled.i`
```

Comment:
Let's avoid having too many styled components that will affect readability. Can we set an CSS rule for `<T.Key>`'s icon in `../../ui-elements/TableKeyValue2` ?

Reply:
done